extra/onnx_ops.py

```diff
@@ -152,8 +152,10 @@ def Softmax_1(input, axis=1): return input.softmax(axis)
 def Softmax_13(input, axis=-1): return input.softmax(axis)
 Softmax = {1: Softmax_1, 13: Softmax_13}   # Softmax default axis changed
 def LogSoftmax(input, axis=-1): return input.log_softmax(axis)
-def Clip(input, min=-3.4e38, max=3.4e38): return input.clip(min, max)
-
+def Clip(input, min=None, max=None):
+  if min is None: min = -3.4e38
```

Comment:
Ugh, really?

Reply:
I know, the problem is that Onnx Model calls the Clip function with an explicit None for min.
```python
# onnx.py - line 88 
# inp -> [<Tensor buffer<60, dtypes.float> on TORCH with grad None>, None, <Tensor buffer<1, dtypes.float> on TORCH with grad None>]
real_fxn(*inp, **opt)
```
That's why it would never default min to -3.4e38 and the test fails without the if.

Reply:
Wait what I'm I doing I can create an elif for this function and do what's needed there.

Reply:
Ahh, I see. Meh, onnx is dumb, will merge.